hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-timelineservice-hbase/hadoop-yarn-server-timelineservice-hbase-client/pom.xml

```diff
@@ -165,6 +165,14 @@
           <groupId>org.mortbay.jetty</groupId>
           <artifactId>jetty-sslengine</artifactId>
         </exclusion>
+        <exclusion>
```

Comment:
Why do you add exclusions ?

Reply:
These dependencies causes issues when starting the Yarn server, but I don't keep the stacktrace. If needed, I can reproduce it.

Reply:
We need to have more logs to understand why there is a need for this exclusion, is it because of the use of HBase 2.1 instead of HBase 2.0? Can you provide the logs which indicate the error if we do not make this exclusion?

Reply:
Do you have any news ?